.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+inherit_gem:
+  rubocop-discourse: default.yml
+inherit_mode:
+  merge:
+    - Exclude
+Discourse/NoChdir:
+  Enabled: false
+RSpec:
+  Enabled: false
```

Comment:
Are those causing issues? 😃

Reply:
Yes because we’re using minitest specs and not RSpec :sweat_smile:

Reply:
ah 😛